src/vos/ilog.c

```diff
@@ -336,6 +339,12 @@ ilog_mag2ver(uint32_t magic) {
 	return (magic & ILOG_VERSION_MASK);
```

Comment:
I guess this works, but just want to make sure if we choose not to shift right 5 bits intentionally or we just forgot to do so.

Reply:
It is intentional as the only requirement from the version is that it differs from the prior one.  I can add a shift though as that might make it easier to follow the debug logs

Reply:
Good to know, thanks.